bindata/assets/kube-controller-manager/svc.yaml

```diff
@@ -14,3 +14,6 @@ spec:
   - name: https
     port: 443
     targetPort: 10257
+  - name: httpscpc
+    port: 10357
```

Comment:
should I choose a different port or create a standalone service?

Reply:
@atiratree double check that with monitoring team, I think you're gonna need a new service or at least a configuration for the 2nd port, since you're not using non-standard 443 here. I think having a service will be simpler.

Reply:
I have switched to using 2 services and 2 service monitors to separate the responsibilities